src/main/webapp/services/assignUserService.js

```diff
@@ -69,25 +72,43 @@ export default class AssignUserService extends BaseService {
   }
 
   openActionDialog(data, service) {
-    service.selectedUser = {};
+    const baseUrl = data.entity.baseUrl;
+    const issueKey = data.entity.issue.key;
 
     const assignTemplate = assignDialog();
     const dialogBuilder = new DialogBuilder('Assign', assignTemplate);
+    let template = null;
+
+    searchIssue(baseUrl, issueKey, service.jwt)
+      .then((issueInfo) => {
+        Object.assign(data, issueInfo.data);
+        service.selectedUser = {};
 
-    const template = service.retrieveTemplate(dialogBuilder, data, service.serviceName);
-    service.openDialog('assignIssue', template.layout, template.data);
+        template = service.retrieveTemplate(dialogBuilder, data, service.serviceName);
+        service.openDialog('assignIssue', template.layout, template.data);
+      })
+      .catch(() => {
+        dialogBuilder.headerError('Couldn\'t fetch issue info');
+        template = service.retrieveTemplate(dialogBuilder, data, service.serviceName);
+        service.openDialog('assignIssue', template.layout, template.data);
+      });
   }
 
   save(data) {
-    if (this.selectedUser.email === undefined) {
-      const assignTemplate = assignDialog();
+    const assignTemplate = assignDialog();
+    const dialogBuilder = new DialogBuilder('Assign', assignTemplate);
 
-      const dialogBuilder = new DialogBuilder('Assign', assignTemplate);
+    if (this.selectedUser.email === undefined) {
       dialogBuilder.error('Please select an user');
 
       const template = this.retrieveTemplate(dialogBuilder, data, this.serviceName);
       this.updateDialog('assignIssue', template.layout, template.data);
     } else {
+      dialogBuilder.loading(true);
+
+      const template = this.retrieveTemplate(dialogBuilder, data, this.serviceName, true);
```

Comment:
I think we should change the button label to 'SAVING' when the attribute loading equals true.

Reply:
Fixed!